Code/GraphMol/FindStereo.cpp

```diff
@@ -225,7 +225,7 @@ StereoInfo getStereoInfo(const Bond *bond) {
           UNDER_CONSTRUCTION("unrecognized bond stereo type");
       }
     } else {
-      sinfo.specified = Chirality::StereoSpecified::Unspecified;
+      UNDER_CONSTRUCTION("unrecognized bond stereo type");
```

Comment:
This change seems wrong; it leads to:
```
In [8]: m = Chem.MolFromSmiles('CC=CC')

In [9]: Chem.FindPotentialStereo(m,cleanIt=True,flagPossible=True)
[06:40:00] 

****
Incomplete Code
This routine is still under development
Violation occurred on line 228 in file /scratch/RDKit_git/Code/GraphMol/FindStereo.cpp
Failed Expression: unrecognized bond stereo type
----------
```
and is responsible for the CI build failures on this PR.
If you revert this change I think the CI tests will all pass (at least it worked on my linux box)

Reply:
Yes this was a mistake.  It has been restored as suggested.